themes/nswds/templates/nswds/Includes/ContentBlocks.ss

```diff
@@ -1,7 +1,7 @@
 <% if $ContentBlocks_Items %>
-<div class="nsw-grid">
+<div class="nsw-grid nsw-grid--spaced">
```

Comment:
Is spaced grid required ?

Reply:
yep, [specifically for content blocks](https://digitalnsw.github.io/nsw-design-system/components/content-block/index.html).